### DISCUSSION TITLE
家人们 nonebot怎么一键删除所有插件

### DISCUSSION THREAD

**Discussion by VeakizZ** [nonebot/discussions#248] [问答]
我插件弄多了好像加载不了

**Comment by Well2333** [comment]
![AXY5E0{_8E(S)I_E4HEKNGH](https://user-images.githubusercontent.com/40534114/202847166-fb8d537b-dcdd-44fb-b057-0eb4db7da26a.jpg)

**Comment by Well2333** [comment]
![QQ图片20221119185518](https://user-images.githubusercontent.com/40534114/202847221-8c8c5e3c-6433-4daf-8372-783553874645.jpg)

**Comment by StarHeartHunt** [comment]
1. 直接重装 Python
2. 使用例如 venv 或者 conda 一类的虚拟 Python 环境管理

或者用 poetry install --sync 同步环境依赖
